src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT04-251-expected.json

```diff
@@ -1,30 +1,24 @@
-﻿{
+{
   "resourceType": "Bundle",
   "type": "batch",
   "entry": [
     {
-      "fullUrl": "urn:uuid:7393bcca-6d2f-543d-7787-0753133a8496",
+      "fullUrl": "urn:uuid:55e2d2b6-8c08-6f34-9f2c-ccf2ba785dba",
```

Comment:
Is this change caused by previous changes? It seems this change won't change the ID.

Reply:
Because the first generation of the ID is very long ago, we have escapped the `MSH` value in subsequent commits.